.travis.yml

```diff
@@ -12,9 +12,6 @@ cache:
 
 matrix:
   include:
-    - php: 5.3
-    - php: 5.4
-    - php: 5.5
     - php: 5.6
     - php: hhvm
     - php: nightly
```

Comment:
This can now be `8.0`

Reply:
`allow_failures` should not contain `8.0` anymore

Reply:
8.0 added already